plugins/inputs/win_services/win_services_test.go

```diff
@@ -39,7 +39,7 @@ type FakeSvcMgr struct {
 	testData testData
 }
 
-func (m *FakeSvcMgr) Disconnect() error {
+func (*FakeSvcMgr) Disconnect() error {
```

Comment:
Probably also doesn't need to be exported?

Reply:
Many methods in this plugin might be unexported. I will be fixing this as part of the work on the `revive:exported` linter.